upstairs/src/downstairs.rs

```diff
@@ -1891,15 +1894,24 @@ impl Downstairs {
         ));
         debug!(self.log, "IO Write {} has deps {:?}", ds_id, dependencies);
 
+        // TODO: can anyone actually give us an empty write?
```

Comment:
I don't think there's anything in the spec that prevents this? Even if there was something, I'd still argue we have to support it anyway.